Index: app/src/main/java/apackage/com/yournight/view/LoginPage.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/apackage/com/yournight/view/LoginPage.java	(revision 842189b0ea84095c659e65a00e21b404e76a9799)
+++ app/src/main/java/apackage/com/yournight/view/LoginPage.java	(revision )
@@ -34,8 +34,6 @@
     private AccessLogin assesLogin;
     private LoginButton loginButton;
     private FacebookNetwork facebook;
-    private Button accCreateButton;
-
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
@@ -44,21 +42,12 @@
         AccessLogin.initialize();
         assesLogin = AccessLogin.getInstance();
         setContentView(R.layout.activity_login_page);
-        accCreateButton = findViewById(R.id.accCreateButton);
         loginButton = findViewById(R.id.facebook_loginButton);
         List<String> fbScope = Arrays.asList("public_profile", "email");
         assesLogin.addSocialNetwork(new FacebookNetwork(this, fbScope));
         facebook = (FacebookNetwork) assesLogin.getSocialNetwork(SocialNetwork.Network.FACEBOOK);
 
         if (facebook.isConnected()){
-            accCreateButton.setText("Weiter");
-            accCreateButton.setOnClickListener(new View.OnClickListener() {
-                @Override
-                public void onClick(View v) {
-                    Intent intent = new Intent(LoginPage.this, SplashScreen.class);
-                    startActivity(intent);
-                }
-            });
             AccessToken token = facebook.getAccessToken();
             assesLogin.setToken(token);
             Log.e("LoginPage", "Facebook AccessLogin(auto) successful with token : " + token.getToken() + "\n" + "and Email : " + token.getEmail());
Index: app/src/main/java/apackage/com/yournight/model/event/EventParser.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/apackage/com/yournight/model/event/EventParser.java	(revision 842189b0ea84095c659e65a00e21b404e76a9799)
+++ app/src/main/java/apackage/com/yournight/model/event/EventParser.java	(revision )
@@ -150,9 +150,13 @@
             /* correct time */
             events[index].getStartTime().split("T");
             String[] eventDate = events[index].getStartTime().split("T");
-            String[] eventTime = eventDate[1].split("\\+");
-            events[index].setStartTime(eventDate[0] + " ab " + eventTime[0]);
-            
+            String[] buffer = eventDate[1].split("\\+");
+            String eventTime = buffer[0];
+            eventTime = eventTime.substring(1, eventTime.length() - 3);
+            events[index].setStartTime(eventDate[0] + " ab " + eventTime + " Uhr");
+
+
+
             Log.e("LOG | ---","----");
             Log.e("LOG | id", events[index].getId());
             Log.e("LOG | name", events[index].getName());
Index: app/src/main/java/apackage/com/yournight/view/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/apackage/com/yournight/view/MainActivity.java	(revision 842189b0ea84095c659e65a00e21b404e76a9799)
+++ app/src/main/java/apackage/com/yournight/view/MainActivity.java	(revision )
@@ -88,11 +88,6 @@
         FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
         transaction.replace(R.id.frame_layout, FragmentHome.newInstance());
         transaction.commit();
-
-        if(userEmail == null || userName == null){
-            loadDataBase();
-        }
-
         //.Remove
         showUserDataInLog();
     }
@@ -104,11 +99,16 @@
     }
 
     private void init(){
+
         sharedPreferences = getPreferences(MODE_PRIVATE);
-        userEmail = asses.getToken().getEmail();
-        userName = asses.getToken().getUserName();
-        userID = asses.getToken().getUserId();
-//        userPictureView.setProfileId(userID);
+        if(userEmail == null || userName == null){
+            loadDataBase();
+        }else {
+            userEmail = asses.getToken().getEmail();
+            userName = asses.getToken().getUserName();
+            userID = asses.getToken().getUserId();
+        //        userPictureView.setProfileId(userID);
+        }
     }
 
     private void init_layout(){
@@ -119,7 +119,7 @@
         Log.e("Main:", userEmail);
         Log.e("Main:", userName);
         Log.e("Main:", userID);
-        Log.e("MainToken:", "" + asses.getToken().getToken());
+//        Log.e("MainToken:", "" + asses.getToken().getToken());
     }
 
 
Index: app/src/main/res/layout/activity_login_page.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/res/layout/activity_login_page.xml	(revision 842189b0ea84095c659e65a00e21b404e76a9799)
+++ app/src/main/res/layout/activity_login_page.xml	(revision )
@@ -11,37 +11,16 @@
     <com.facebook.login.widget.LoginButton
         android:id="@+id/facebook_loginButton"
         android:layout_width="250dp"
-        android:layout_height="40dp"
+        android:layout_height="50dp"
         android:layout_gravity="center_horizontal"
-        android:layout_marginEnd="8dp"
-        android:layout_marginStart="8dp"
+        android:layout_marginBottom="250dp"
+        android:layout_marginEnd="25dp"
+        android:layout_marginStart="25dp"
         android:layout_marginTop="250dp"
+        app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.51"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-    <TextView
-        android:id="@+id/textView"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginEnd="16dp"
-        android:layout_marginStart="275dp"
-        android:layout_marginTop="24dp"
-        android:text="@string/einloggen"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
-
-    <Button
-        android:id="@+id/accCreateButton"
-        android:layout_width="258dp"
-        android:layout_height="51dp"
-        android:layout_marginEnd="8dp"
-        android:layout_marginStart="8dp"
-        android:layout_marginTop="24dp"
-        android:text="@string/account_erstelln"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/facebook_loginButton" />
-
 </android.support.constraint.ConstraintLayout>
Index: app/src/main/java/apackage/com/yournight/view/SplashScreen.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/apackage/com/yournight/view/SplashScreen.java	(revision 842189b0ea84095c659e65a00e21b404e76a9799)
+++ app/src/main/java/apackage/com/yournight/view/SplashScreen.java	(revision )
@@ -1,12 +1,9 @@
 package apackage.com.yournight.view;
 
 import android.content.Intent;
-import android.content.res.ColorStateList;
-import android.graphics.Color;
 import android.os.Handler;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
-import android.view.Menu;
 import android.view.View;
 import android.view.animation.Animation;
 import android.view.animation.AnimationUtils;
@@ -19,14 +16,12 @@
 
 import apackage.com.yournight.R;
 import apackage.com.yournight.model.Location.GeoOperations;
-import apackage.com.yournight.model.Location.Location;
-import apackage.com.yournight.model.access.AzureAccessToken;
 import apackage.com.yournight.model.network.EventNetwork;
 
 public class SplashScreen extends AppCompatActivity implements Animation.AnimationListener {
 
     /** Duration of wait **/
-    private final int SPLASH_DISPLAY_LENGTH = 5000;
+    private final int SPLASH_DISPLAY_LENGTH = 8000;
     private RequestQueue requestQueue;
     private EventNetwork server;
     private TextView tv;
@@ -36,19 +31,17 @@
     private static final int NUMBER_OF_ANIMATIONS = 2;
     private int animationFinishedCount = 0;
     /** Called when the activity is first created. */
+
     @Override
     public void onCreate(Bundle icicle) {
-
         super.onCreate(icicle);
         setContentView(R.layout.activity_splash_screen);
-        startAnimation();
-
         // GeoOperations geoOperations = new GeoOperations(this, this);
         GeoOperations geoOperations = null;
         requestQueue = Volley.newRequestQueue(this);
         EventNetwork eventNetwork = new EventNetwork(requestQueue, geoOperations);
         eventNetwork.getEventsFromServer();
-
+        startAnimation();
         /* New Handler to start the Menu-Activity
          * and close this Splash-Screen after some seconds.*/
         new Handler().postDelayed(new Runnable(){
@@ -58,12 +51,10 @@
                 Intent mainIntent = new Intent(SplashScreen.this,MainActivity.class);
                 SplashScreen.this.startActivity(mainIntent);
                 SplashScreen.this.finish();
-
             }
         }, SPLASH_DISPLAY_LENGTH);
-
-
     }
+
     public void startAnimation() {
         tv = (TextView) findViewById(R.id.splashtext);
         iv = (ImageView) findViewById(R.id.splashscreen);
@@ -77,6 +68,7 @@
         tv.startAnimation(downsideUp);
         iv.startAnimation(upsideDown);
     }
+
     @Override
     public void onAnimationStart(Animation animation) {
         //Nothing
@@ -91,7 +83,6 @@
         }
 
     }
-
     @Override
     public void onAnimationRepeat(Animation animation) {
         //Nothing
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/modules.xml	(revision 842189b0ea84095c659e65a00e21b404e76a9799)
+++ .idea/modules.xml	(revision )
@@ -3,8 +3,7 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
-      <module fileurl="file://C:\Users\Gobi\Mobile Computing\WorkspaceAndroid\yournightTest\yournight.iml" filepath="C:\Users\Gobi\Mobile Computing\WorkspaceAndroid\yournightTest\yournight.iml" />
-      <module fileurl="file://$PROJECT_DIR$/.idea/yournightTest.iml" filepath="$PROJECT_DIR$/.idea/yournightTest.iml" />
+      <module fileurl="file://$PROJECT_DIR$/yournight.iml" filepath="$PROJECT_DIR$/yournight.iml" />
     </modules>
   </component>
 </project>
\ No newline at end of file
Index: app/src/main/java/apackage/com/yournight/view/fragment/FragmentHome.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/apackage/com/yournight/view/fragment/FragmentHome.java	(revision 842189b0ea84095c659e65a00e21b404e76a9799)
+++ app/src/main/java/apackage/com/yournight/view/fragment/FragmentHome.java	(revision )
@@ -43,7 +43,7 @@
 
     private RecyclerViewAdapter adapter;
 
-    private final static int MAXEVENTS = 5;
+    private final static int MAXEVENTS = 10;
 
     public void setRecyclerView(View view){
 
@@ -55,13 +55,18 @@
         ArrayList<String> descriptions = new ArrayList<>();
 
 
-        if(events != null){
-
-            for(int i = 0; i <= MAXEVENTS; i++){
+        if(events != null) {
+            for (int i = 0; i <= MAXEVENTS; i++) {
                 bitmaps.add(icon);
                 titles.add(events[i].getName());
                 descriptions.add(events[i].getStartTime());
             }
+        }else{
+            for (int i = 0; i <= MAXEVENTS; i++) {
+                bitmaps.add(icon);
+                titles.add("Network error");
+                descriptions.add("Network error");
+            }
         }
 
 //
Index: app/src/main/java/apackage/com/yournight/model/network/EventNetwork.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/apackage/com/yournight/model/network/EventNetwork.java	(revision 842189b0ea84095c659e65a00e21b404e76a9799)
+++ app/src/main/java/apackage/com/yournight/model/network/EventNetwork.java	(revision )
@@ -22,7 +22,6 @@
 
 import apackage.com.yournight.model.Location.GeoOperations;
 import apackage.com.yournight.model.Location.Location;
-import apackage.com.yournight.model.access.AzureAccessToken;
 import apackage.com.yournight.model.event.Event;
 import apackage.com.yournight.model.event.EventParser;
 
@@ -46,8 +45,6 @@
     private GeoOperations geoOperations;
     private Location location;
     private JSONObject sendJSONObject;
-    /* Events */
-
 
     /* Constructor */
     public EventNetwork(RequestQueue requestQueue, GeoOperations geoOperations) {
@@ -121,7 +118,7 @@
                 return headers;
             }
         };
-        jsonObjectRequest.setRetryPolicy(new DefaultRetryPolicy(10000,
+        jsonObjectRequest.setRetryPolicy(new DefaultRetryPolicy(30000,
                 DefaultRetryPolicy.DEFAULT_MAX_RETRIES,
                 DefaultRetryPolicy.DEFAULT_BACKOFF_MULT));
 
